client/blockchain.go

```diff
@@ -20,6 +20,7 @@ const BlockchainTypeEVM = "evm"
 // of network types within the test suite
 type BlockchainClient interface {
 	Get() interface{}
+	CalculateTxGasCostWithBuffer(maxGasUsedValue *big.Int) (*big.Float, error)
```

Comment:
`CalculateTxGasCostWithBuffer` seems needlessly wordy, and implies that there is a method `CalculateTxGasCost**Without**Buffer`.

A better name I think is `CalculateTxGas`